lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart

```diff
@@ -28,7 +30,56 @@ class MyUserPageViewModel {
   final UserModelChanger userChanger;
   final AccountService accountService;
 
-  Future<bool> acceptBid(List<BidIn> bidIns, BuildContext context) async {
+  Future<String?> setFirst(BidIn bidIn, BuildContext context) async {
+    String? addressOfUserB;
+    if (bidIn.public.speed.num != 0) {
+      Map sessionWithAddress = await accountService.getAllWalletAddress();
+      List<String> addresses = [];
+      for (List<String> val in sessionWithAddress.values) {
+        addresses.addAll(val);
+      }
+      if (addresses.isNotEmpty) {
+        addressOfUserB = addresses.first;
+      }
+    }
+    return addressOfUserB;
+  }
+
+  Future acceptBid(List<BidIn> bidIns, BuildContext context) async {
+    String? addressOfUserB;
+    BidIn bidIn = bidIns.first;
+    if (!(bidIn.public.active)) {
+      CustomAlertWidget.showToastMessage(context, "Bidder is not active");
```

Comment:
this should never happen ~ we should never be showing non-active bid to Host
but, ok, can leave for now